chrome_extension/vendor/mooltipass/mcCombinations.js

```diff
@@ -140,6 +140,15 @@ var extendedPost = {
 	}
 }
 
+var extendedSubmitButton = {
+	'vodafone.de': function(form) {
+		return form.parent().parent().find('#loginBtn')
```

Comment:
that would be great thanks!
how can this form be submitted if the submit is out of the container? through some JS?